src/app/components/core/configuration-manager/view-config-item/view-config-item.component.html

```diff
@@ -27,7 +27,7 @@
                           [disabled]="configItem?.value?.hasOwnProperty('editable') && !configItem?.value?.editable" />
                       </label>
                     </ng-container>
-                    <div class="field">
+                    <div class="field" [style.max-width.px]=cdmrTextareaWidth id="cdmr-field" [ngClass]="{'codemirror-field': getConfigAttributeType(configItem.value.type) === 'LONG_TEXT' || getConfigAttributeType(configItem.value.type) === 'RAW_TEXT' || getConfigAttributeType(configItem.value.type) === 'SCRIPT'}">
```

Comment:
avoid ID? e.g. in python35 filter (and/or + South plugins with such fields) will cause multiple objects with same ID in DOM.